src/lang/english.txt

```diff
@@ -1553,8 +1553,11 @@ STR_CONFIG_SETTING_ENDING_YEAR                                  :Scoring end yea
 STR_CONFIG_SETTING_ENDING_YEAR_HELPTEXT                         :Year the game ends for scoring purposes. At the end of this year, the company's score is recorded and the high-score screen is displayed, but the players can continue playing after that.{}If this is before the starting year, the high-score screen is never displayed.
 STR_CONFIG_SETTING_ENDING_YEAR_VALUE                            :{NUM}
 STR_CONFIG_SETTING_ENDING_YEAR_ZERO                             :Never
-STR_CONFIG_SETTING_SMOOTH_ECONOMY                               :Enable smooth economy (more, smaller changes): {STRING2}
-STR_CONFIG_SETTING_SMOOTH_ECONOMY_HELPTEXT                      :When enabled, industry production changes more often, and in smaller steps. This setting has usually no effect, if industry types are provided by a NewGRF
+STR_CONFIG_SETTING_ECONOMY_TYPE                                 :Economy type: {STRING2}
+STR_CONFIG_SETTING_ECONOMY_TYPE_HELPTEXT                        :Smooth economy makes production changes more often, and in smaller steps. Stable stops production changes and industry closures. This setting has usually no effect, if industry types are provided by a NewGRF.
```

Comment:
`This setting has usually no effect, if industry types are provided by a NewGRF.`
pretty sure the comma is wrong here.
`This setting usually has no effect if industry types are provided by a NewGRF.`